test/tools/isoltest.cpp

```diff
@@ -98,6 +100,28 @@ class TestTool
 string TestTool::editor;
 bool TestTool::m_exitRequested = false;
 
+void TestTool::runEditor(const fs::path& filename)
+{
+#if defined(_WIN32)
+	std::string path = filename.string();
+	SHELLEXECUTEINFO ShExecInfo = { 0 };
+	ShExecInfo.cbSize = sizeof(SHELLEXECUTEINFO);
```

Comment:
This all looks sooo scary...

Reply:
Not more scary than the already existing `system()`.

Reply:
touché